compactor/main.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
We can remove the compactor code as we are using sed within inject-manifests.sh to accomplish the same thing.